emission/core/wrapper/suggestion_sys.py

```diff
@@ -91,10 +98,9 @@ def request(host, path, api_key, url_params=None):
         'Authorization': 'Bearer %s' % api_key,
     }
 
-    print(u'Querying {0} ...'.format(url))
+    # print(u'Querying {0} ...'.format(url))
 
     response = requests.request('GET', url, headers=headers, params=url_params)
```

Comment:
Can remove the TODO about `urllib` given that you have already moved to `requests`?